.macos

```diff
@@ -38,10 +38,7 @@ echo "brew installing stuff"
 # bat: A cat(1) clone with syntax highlighting and Git integration.
 # switchaudio-osx: allows me to switch the audio device via the command line
 brew install git hub libdvdcss ripgrep imagemagick watchman tree bat \
-delta switchaudio-osx
-
-echo "installing node (via n-install)"
-curl -L https://git.io/n-install | bash
+delta switchaudio-osx node
```

Comment:
I prefer installing node this way

```suggestion
delta switchaudio-osx

echo "installing node (via n-install)"
curl -L https://git.io/n-install | bash
```

Reply:
Any specific reason for that? 🤔

Reply:
I like using https://npm.im/n to manage my installed node version